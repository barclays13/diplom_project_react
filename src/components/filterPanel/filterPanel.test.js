import React from 'react';
import FilterPanel from './filterPanel';
import {shallow} from 'enzyme';

describe('Testing <FilterPanel>', () => {
    it ('FilterPanel have renderer correctly', () => {
    const filter = shallow(<FilterPanel/>);
    expect(filter).toMatchSnapshot();
    })
});