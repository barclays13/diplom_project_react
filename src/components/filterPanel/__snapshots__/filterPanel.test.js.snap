// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <FilterPanel> FilterPanel have renderer correctly 1`] = `
<div
  className="btn-group"
>
  <button
    className="shop__filter-btn"
    onClick={[Function]}
    type="button"
  >
    All
  </button>
  <button
    className="shop__filter-btn"
    onClick={[Function]}
    type="button"
  >
    Brazil
  </button>
  <button
    className="shop__filter-btn"
    onClick={[Function]}
    type="button"
  >
    Kenya
  </button>
  <button
    className="shop__filter-btn"
    onClick={[Function]}
    type="button"
  >
    Columbia
  </button>
</div>
`;
